misk-hibernate/src/main/kotlin/misk/vitess/StartVitessDaemon.kt

```diff
@@ -0,0 +1,42 @@
+package misk.vitess
+
+import com.github.dockerjava.api.DockerClient
+import com.github.dockerjava.core.DockerClientBuilder
+import com.github.dockerjava.netty.NettyDockerCmdExecFactory
+import com.squareup.moshi.Moshi
+import misk.jdbc.DataSourceConfig
+import misk.resources.ResourceLoader
+import kotlin.reflect.KClass
+
+/**
+ * A helper method to start the vitess cluster outside of the dev server or test process, to enable
+ * rapid iteration. This should be called directly a `main()` function, for example:
+ *
+ *  fun main() {
+ *    val config = MiskConfig.load<MyAppConfig>("myapp")
+ *    startVitessDaemon(MyAppDb::class, config.data_source_clusters.values.first().writer)
+ *  }
+ *
+ */
+fun startVitessDaemon(
```

Comment:
nit: I reckon you can put this in the "companion" object inside `StartVitessService`. It doesn't quite feel like something small like this deserves an entire file all on its own. :-)